### ISSUE TITLE
AttributeError: 'Literal' object has no attribute 'build'

### ISSUE THREAD

**Issue by gnbl** [evtn/soda#6]
Attempting the example in https://github.com/evtn/soda#text

```
from soda import Tag, Root, Literal
svg = Root()(
    Tag.g(Literal('<path d="M0 0 L10 0 Z"/>', escape=False))
).render()

```

**Comment by gnbl** [comment]
Temporary fix:

    def build(self, *args):
        return self.children[0]

**Comment by evtn** [comment]
> Temporary fix:
> 
> ```
> def build(self, *args):
>     return self.children[0]
> ```

That doesn't seem right, `Tag.build` should return a generator
I'll check in a moment

**Comment by evtn** [comment]
A proper `Literal.build` should be

```python

def build(self, tab_size: int = 0, tab_level: int = 0) -> Generator[str, None, None]:
    separator = "\n" * bool(tab_size)

    for child in self.children:
        yield separator
        yield from self.build_child(child, tab_size, tab_level + 1)
        
```

I will add this in 1.0.1

**Comment by gnbl** [comment]
AttributeError: 'Literal' object has no attribute 'build_child'

**Comment by evtn** [comment]
> ```
> AttributeError: 'Literal' object has no attribute 'build_child'
> ```

Yes, this won't work on old vesrions without another edit
`Literal` should extend `Tag`
`class Literal(Tag):`

**Comment by gnbl** [comment]
No worries, take your time.

**Comment by evtn** [comment]
I took my time and fixed this in 1.0.1, which should be available already.

**Comment by gnbl** [comment]
`escape=False` does not appear to have an effect.

**Comment by evtn** [comment]
And that one was fixed in 1.0.2

Seems like I didn't use `Literal` enough to catch obvious bugs
